services/web/server/src/simcore_service_webserver/projects/_crud_handlers_models.py

```diff
@@ -150,3 +150,19 @@ class Config:
 
 class ProjectActiveParams(BaseModel):
     client_session_id: str
+
+
+class ProjectListFullSearchParams(PageQueryParameters):
+    text: str | None = Field(
+        default=None,
+        description="Multi column full text search, across all folders and workspaces",
+        max_length=100,
+        example="My Project",
+    )
+
+    @validator("text", pre=True)
+    @classmethod
+    def text_check_empty_string(cls, v):
```

Comment:
MINOR: since all these validators are never explicitly called, I start to name them as protected i.e. `_`. This way it is easy to spot better members that are from the public interface e.g. the auto-completion will not show you validators